### DISCUSSION TITLE
What is the difference between Exemplar and Sample?

### DISCUSSION THREAD

**Discussion by Tusenka** [prometheus/prometheus#14999] [Q&A]
What is the difference between Exemplar and Sample?

In both cases it is timestamp/value for a specific series

https://pkg.go.dev/github.com/prometheus/prometheus/tsdb/record#RefExemplar https://pkg.go.dev/github.com/prometheus/prometheus/tsdb/record#RefSample

**Comment by bboreham** [comment]
https://prometheus.io/docs/concepts/data_model/#samples
https://prometheus.io/docs/prometheus/latest/feature_flags/#exemplars-storage

Exemplars have their own labels
